<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetVehicleLight</class>
 <widget class="QWidget" name="SetVehicleLight">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: rgb(51,153,255);
}

QLabel
{
background-color:rgb(51,153,255);
border:0px;
font:25px &quot;SimHei&quot;;
color:white;
}

QPushButton
{
font:25px, &quot;SimHei&quot;;
outline:none;
border-image: url(:/image/buttonUP.jpg);
color:rgb(0, 32, 96);
border-radius:2px;
}

QPushButton:disabled
{
outline:none;
border:1px solid white;
border-image: url(:/image/buttonUP.jpg);
font:25px, &quot;SimHei&quot;;
color:rgb(180, 180, 180);
}</string>
  </property>
  <widget class="QPushButton" name="btnPlus">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>180</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>＋</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSet">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>30</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>亮度调节</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnModeAuto">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid white;
border-image: url(:/image/buttonUP.jpg);
font:25px, &quot;SimHei&quot;;
color:rgb(0, 32, 96);
border-radius:2px;</string>
   </property>
   <property name="text">
    <string>自动</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_set_value">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>180</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelMode">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>模式</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMinus">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>180</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>－</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnModeManu">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid white;
border-image: url(:/image/buttonUP.jpg);
font:25px, &quot;SimHei&quot;;
color:rgb(0, 32, 96);
border-radius:2px;</string>
   </property>
   <property name="text">
    <string>手动</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>390</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid white;
border-image: url(:/image/buttonUP.jpg);
font:25px, &quot;SimHei&quot;;
color:rgb(0, 32, 96);
border-radius:2px;</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSet_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>240</y>
     <width>451</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:20px;
color:red;</string>
   </property>
   <property name="text">
    <string>仅在手动模式下亮度调节可用，调节范围0.1～1.0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
